### PR TITLE
Roslyn 64bit

### PR DESCRIPTION
**Pull request by iainx**
64bit patches

### CODE REVIEW

File: version-checks [hunk 1]
```diff
@@ -17,8 +17,8 @@ DEP[0]=md-addins
 DEP_NAME[0]=MDADDINS
 DEP_PATH[0]=${top_srcdir}/../md-addins
 DEP_MODULE[0]=git@github.com:xamarin/md-addins.git
-DEP_NEEDED_VERSION[0]=3d22dbe74dd0cef0aa30ee573e1bbf2b8555b2e2
-DEP_BRANCH_AND_REMOTE[0]="roslyn origin/roslyn"
+DEP_NEEDED_VERSION[0]=eb4701b84a5d481d2ada0f52adf231e395ffcbaa
+DEP_BRANCH_AND_REMOTE[0]="roslyn origin/roslyn-64bit"
```
Comment (DavidKarlas) [inline version-checks]: PR probably shouldn't have "-64bit" branch name change for md-addins included

File: main/Makefile.am [hunk 2]
```diff
@@ -18,6 +18,8 @@ all-local: vcrevision restore-packages sln_build
 clean: clean-local
 
 clean-local: sln_clean
+	cd external && $(MAKE) clean
```
Comment (Therzok) [inline main/Makefile.am]: This should be backported to master.

File: main/build/MacOSX/Makefile.am [hunk 3]
```diff
@@ -31,12 +32,12 @@ dmg: render.exe app
 	./make-dmg-bundle.sh
 
 monostub: monostub.m $(MONOSTUB_EXTRA_SOURCES)
-	gcc -Wall -mmacosx-version-min=10.10 -m32 -o $@ $^ -framework AppKit
-#	gcc -Wall -mmacosx-version-min=10.10 -m32 -o $@ $^ -framework AppKit -isysroot $(SDK_PATH)
+	gcc -g -Wall -mmacosx-version-min=10.10 -m$(MONOSTUB_ARCH) -o $@ monostub.m -framework AppKit
```
Comment (Therzok) [inline main/build/MacOSX/Makefile.am]: Do we want the shipped monostub to have debug symbols?

File: version-checks (hunk 1 above)
Comment (iainx) [inline version-checks]: Yeah, I was going to fix that up once the md-addins PR was merged

File: main/build/MacOSX/Makefile.am (hunk 3 above)
Comment (iainx) [inline main/build/MacOSX/Makefile.am]: Probably not, you're right

**Pull request merged**